generators/app/index.js

```diff
@@ -116,11 +116,11 @@ exports['default'] = _yeomanGenerator.Base.extend({
 
   configuring: {
     deps: function deps() {
-      this.getPackageVersions('deps', ['babel-core', 'es6-promise', 'whatwg-fetch', 'lodash', ['react', '0.14.0'], ['react-dom', '0.14.0'], ['redux', '3.0.4'], ['react-redux', '4.0.0'], ['redux-devtools', '2.1.5'], ['redux-thunk', '1.0.0']]);
+      this.getPackageVersions('deps', [['babel-core', '5.8.33'], 'es6-promise', 'whatwg-fetch', 'lodash', ['react', '0.14.0'], ['react-dom', '0.14.0'], ['redux', '3.0.4'], ['react-redux', '4.0.0'], ['redux-devtools', '2.1.5'], ['redux-thunk', '1.0.0']]);
     },
 
     devDeps: function devDeps() {
-      this.getPackageVersions('devDeps', ['webpack', 'webpack-dev-server', 'css-loader', 'babel-core', 'babel-loader', 'react-hot-loader', 'style-loader', 'extract-text-webpack-plugin', 'cssnext-loader']);
+      this.getPackageVersions('devDeps', ['webpack', 'webpack-dev-server', 'css-loader', ['babel-core', '5.8.33'], 'babel-loader', 'react-hot-loader', 'style-loader', 'extract-text-webpack-plugin', 'cssnext-loader']);
```

Comment:
You should also make this change in `index.babel.js` as well to make sure this doesn't get overwritten on next change. Also pin down `babel-loader` to `5.0.0` while you're in there, since I was still getting 6.x for that with this change.

And a note that you can use `5.0.0` for all the babel deps and it will install the latest in the 5.x.x range without needing to look up exact versions. It will function exactly the same as you have it here, but just a tip to make it easier.